old-simulator-doc.txt

```diff
@@ -68,21 +68,21 @@ foeModifyMove(user, target, move) [on target]
 
 	examples: [ability] Adaptability, [ability, foe ability] No Guard
 
-modifyPokemon(pokemon) [on pokemon]
-	Any effect that modifies a pokemon should be applied here.
+modifyPokémonpokemon
+effect that modifies a Pokémon should be applied here.
 
-	These are the properties of pokemon you can modify:
+	These are the properties of Pokémon you can modify:
 	- pokemon.stats
 	- pokemon.weightkg
 	- pokemon.type
 
-	Whenever any effect is added or removed, the pokemon is reset and
-	modifyPokemon is fired.
+	Whenever any effect is added or removed, the Pokémon is reset and
+	modifypokemon is fired.
 
 	examples: [ability] Light Metal, [volatile] Conversion 2, etc
 
-stats(stats, pokemon) [on pokemon]
-	A shortcut event for modifyPokemon(pokemon) : stats = pokemon.stats.
+stats(stats, pokemon) [on Pokémon]
+	A shortcut event for modifypokemon(pokemon) : stats = pokemon.stats.
```

Comment:
This is wrong though? What were you trying to do here?

Reply:
Whoops I meant to change the "Pokémon" in the function commands back to "pokemon". I must have gotten a little confused.

Sorry about that, I would like to help fix some bugs here but am still new to coding.